lib/mayaUsd/resources/scripts/mayaUsdDuplicateAsUsdDataOptions.py

```diff
@@ -150,22 +149,11 @@ def _fillDuplicateAsUsdDataOptionsDialog(subLayout, optionsText, action):
         mayaUsdTranslatorExport("{subLayout}", "{action}=all;!output", "{optionsText}", "")
         '''.format(optionsText=optionsText, subLayout=subLayout, action=action))
 
-
-def _hasDuplicateAsUsdDataOptionsHelp():
```

Comment:
I removed this method as this doesn't affect the localization. All the content ids will exist. Moreover, this type of check was only used in two places.